src/Breadcrumb/index.jsx

```diff
@@ -12,20 +11,20 @@ const Breadcrumbs = ({
 
   return (
     <nav aria-label="breadcrumb">
-      <ol className={classNames(styles['list-inline'])}>
+      <ol className={classNames('list-inline')}>
```

Comment:
Why not just ```<ol className="list-inline">```?

Reply:
It should be. You're right let's get this cleaned up in subsequent improvements.